packages/rest/src/__tests__/unit/rest.server/rest.server.open-api-spec.unit.ts

```diff
@@ -375,6 +386,26 @@ describe('RestServer.getApiSpec()', () => {
     expect(spec.info).to.eql(EXPECTED_SPEC_INFO);
   });
 
+  context('OpenApiSpecOptions', () => {
+    it('disables consolidator if disableOASConsolidator is set to true', async () => {
+      const options: {rest: RestServerConfig} = {
+        rest: {openApiSpec: {disableOASConsolidator: true}},
```

Comment:
Have you considered using a shorter flag name? In my experience, the best names are using the point of view of the consumer (what are they trying to accomplish using the flag), not the particular implementation details (how is the flag implemented). In that light, I am proposing to use the name `consolidate` with the default value `true`.

```suggestion
        rest: {openApiSpec: {consolidate: false}},
```

Thoughts?

Reply:
> what are they trying to accomplish using the flag

I.e. the user wants to consolidate the spec, or disable consolidation.

> how is the flag implemented

I.e. OASConsolidator is invoked or skipped